server/routes/delius.ts

```diff
@@ -19,14 +20,18 @@ export default function deliusRoutes(router: Router, services: Services) {
     allowEmptyQuery: true,
   })
   router.post('/delius/nationalSearch', deliusSearch.post)
-  router.get('/delius/nationalSearch', deliusSearch.get, (req, res) =>
-    res.render('pages/deliusSearch/index', {
-      deliusUrl: config.delius.url,
-      sentry: config.sentry,
-      ...(res.locals.searchResponse
-        ? mapResults(res.locals.searchResponse, res.locals.searchRequest, req.user.username)
-        : {}),
-    }),
+  router.get(
+    '/delius/nationalSearch',
+    deliusSearch.get,
+    (req, res, next) => hmppsAudit(req, res, next),
+    (req, res) =>
```

Comment:
You should be able to just pass in a reference to the function here:
```suggestion
    deliusSearch.get,
    hmppsAudit,
    (req, res) =>
```